relayer-cli/src/commands/tx/client.rs

```diff
@@ -208,7 +208,18 @@ impl Runnable for TxUpgradeClientCmd {
         let client = ForeignClient::find(src_chain, dst_chain, &self.client_id)
             .unwrap_or_else(exit_with_unrecoverable_error);
 
-        let outcome = client.upgrade();
+        // Assumption: this query is run while the chain is halted as a result of an upgrade
```

Comment:
Consider outputting an `info` or `warn` here saying that Hermes is using `src_upgrade_height` and it assumes chain `src_chain` is halted for update at that height.

Reply:
This is only meant as a temporary hack until we introduce the `--upgrade_height` flag (which IMO should be before v1.0). See [this comment](https://github.com/informalsystems/ibc-rs/pull/2304/files#r899070005).

Reply:
[83d6396](https://github.com/informalsystems/ibc-rs/pull/2304/commits/83d6396c5b34fe92d17088fce1ad4067e7dd907f)